packages/core/src/types/oas2.ts

```diff
@@ -330,32 +320,48 @@ const SecurityScheme: NodeType = {
     scopes: { type: 'object', additionalProperties: { type: 'string' } },
   },
   required(value) {
-    if (!value?.type) {
-      return ['type'];
+    switch (value?.type) {
```

Comment:
`in: { type: 'string', enum: ['query', 'header', 'cookie'] }`=>in: { type: 'string', enum: ['query', 'header'] }